libnavigation-core/src/main/java/com/mapbox/navigation/core/routealternatives/RouteAlternativesCacheManager.kt

```diff
@@ -21,14 +22,24 @@ internal class RouteAlternativesCacheManager(
         }
     }
 
+    @VisibleForTesting
+    internal companion object {
+        internal const val CACHE_SIZE = 3
+    }
```

Comment:
You wouldn't need to make it visible for testing if it was a constructor parameter:
```kotlin
internal class RouteAlternativesCacheManager(
    navigationSession: NavigationSession,
    cacheSize: Int,
)
```

Reply:
I wouldn't provide an option to setup cache size outside